frontend/app/src/components/display/BalanceDisplay.vue

```diff
@@ -76,7 +76,7 @@ const valueInCurrency = computed(() => {
       'text-rui-error': mode === 'loss'
     }"
   >
-    <div :class="`d-flex flex-column align-${align}`">
+    <div :class="`flex flex-col align-${align}`">
```

Comment:
Yes this here I noticed now but let's not do this 

https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Reply:
a little hindsight, updated now